Lib.Web.Mvc/JQuery/JqGrid/DataAnnotations/JqGridColumnEditableAttribute.cs

```diff
@@ -274,5 +273,21 @@ protected override void InternalOnMetadataCreated(ModelMetadata metadata)
             metadata.SetColumnFormOptions(FormOptions);
         }
         #endregion
+
+        #region Overrides of JqGridColumnElementAttribute
+
+        /// <summary>
+        /// When overriden in delivered class, provides a dictionary where keys should be valid attributes for the element.
+        /// </summary>
+        protected override IDictionary<string, object> HtmlAttributes
```

Comment:
This property is intended to be overridden by the library users, the way this has been implemented will make them unintentionally break the "Multiple" function without them even knowing.